documentation/Rotational Motion.tex

```diff
@@ -0,0 +1,187 @@
+
+The first derivative is:
+\begin{align}
+\deriv{t}{}\frac{\vV}{\norm\vV} &= \frac{\norm\vV \dot{\vV} - \deriv{t}{\norm\vV} \vV}{\norm\vV^2} \nonumber \\
+&=\frac{\norm\vV \dot{\vV} - \frac{\pascal{\scal{\vV}{\dot{\vV}}}}{\norm\vV} \vV}{\norm\vV^2} \nonumber \\
+&=\frac{\dot{\vV}}{\norm\vV} - \vV \frac{\pascal{\scal{\vV}{\dot{\vV}}}}{\norm\vV^3}
+\label{eqndv}
+\end{align}
+
+The second derivative is somewhat more complicated:
+\begin{align}
+\deriv[2]{t}{}\frac{\vV}{\norm\vV} &=\deriv{t}{}\pa{\frac{\norm\vV^2 \dot{\vV} - \pascal{\scal{\vV}{\dot{\vV}}} \vV}{\norm\vV^3}} \nonumber \\
+&=\frac{\norm\vV^3 \deriv{t}{}\pa{\norm\vV^2 \dot{\vV} - \pascal{\scal{\vV}{\dot{\vV}}} \vV} - 
+  3 \norm\vV \pascal{\scal{\vV}{\dot{\vV}}}\pa{\norm\vV^2 \dot{\vV} - \pascal{\scal{\vV}{\dot{\vV}}} \vV}}{\norm\vV^6} \nonumber \\
+&=\frac{2 \pascal{\scal{\vV}{\dot{\vV}}} \dot{\vV} +
+    \norm\vV^2 \ddot{\vV} -
+    \pa{\norm{\dot{\vV}}^2 + \pascal{\scal{\vV}{\ddot{\vV}}}} \vV -
+    \pascal{\scal{\vV}{\dot{\vV}}} \dot{\vV}}
+    {\norm\vV^3} -
+  3\frac{\norm\vV^3 \pascal{\scal{\vV}{\dot{\vV}}} \dot{\vV} - \norm\vV \pascal{\scal{\vV}{\dot{\vV}}}^2 \vV}
+    {\norm\vV^6} \nonumber \\
+&=\frac{\ddot{\vV}}{\norm\vV} - 2 \dot{\vV} \frac{\pascal{\scal{\vV}{\dot{\vV}}}}{\norm\vV^3} 
+  - \vV \frac{\norm{\dot{\vV}}^2 + \pascal{\scal{\vV}{\ddot{\vV}}}}{\norm\vV^3} + 3 \vV \frac{\pascal{\scal{\vV}{\dot{\vV}}}^2}{\norm\vV^5}
+\label{eqnddv}
+\end{align}
+
+\section*{Angular velocity}
+To compute the angular velocity, we start by deriving the vectors (\ref{eqnFBN}) and obtain:
+\begin{equation}
+\begin{dcases}
+\dot{\vF} &= \dot{\vr} \\
+\dot{\vB} &= \vr \wedge \ddot{\vr} \\
+\dot{\vN} &= \dot{\vB} \wedge \vF + \vB \wedge \dot{\vF}
+\end{dcases}
+\label{eqndFBN}
+\end{equation}
+Injecting these expressions in the derivative formula (\ref{eqndv}) makes it possible to compute the trihedron of the derivatives $\pa{\dot{\vf}, \dot{\vn}, \dot{\vb}}$ of (\ref{eqnfbn}).  The angular velocity is then written:
+\[
+\VectorSymbol{\gw} = \pascal{\scal{\dot{\vn}}{\vb}} \vf + \pascal{\scal{\dot{\vb}}{\vf}} \vn + \pascal{\scal{\dot{\vf}}{\vn}} \vb
+\]
+
+\section*{Angular acceleration}
+To compute the angular acceleration, we start by deriving the vectors (\ref{eqndFBN}) and obtain:
+\begin{equation}
+\begin{dcases}
+\ddot{\vF} &= \ddot{\vr} \\
+\ddot{\vB} &= \dot{\vr} \wedge \ddot{\vr} + \vr \wedge \vr^{(3)} \\
+\ddot{\vN} &= \ddot{\vB} \wedge \vF + 2 \dot{\vB} \wedge \dot{\vF} + \vB \wedge \ddot{\vF} \\
+\end{dcases}
+\label{eqnddFBN}
+\end{equation}
+Injecting these expressions in the second derivative formula (\ref{eqnddv}) makes it possible to compute the trihedron of the second derivatives $\pa{\ddot{\vf}, \ddot{\vn}, \ddot{\vb}}$ of (\ref{eqnfbn}).  The angular acceleration is then written:
+\begin{align*}
+\dot{\VectorSymbol{\gw}} &= \pascal{\scal{\ddot{\vn}}{\vb}} \vf + \pascal{\scal{\dot{\vn}}{\dot{\vb}}} \vf + \pascal{\scal{\dot{\vn}}{\vb}} \dot{\vf} \\
```

Comment:
Document the derivation of 𝝎̇.

Reply:
The derivation of 𝝎̇ is rather obvious once the formula for 𝝎 is known.  It's just the formula for the derivative of products.